mne/io/base.py

```diff
@@ -2058,3 +2059,190 @@ def _check_update_montage(info, montage):
                        "because they are EOG channels use the eog parameter."
                        % str(missing_positions))
                 raise KeyError(err)
+
+
+def read_raw(input_fname, montage=None, eog=[], misc=[], reference=None,
+             params=None, preload=False, verbose=None):
```

Comment:
So you can just put `**kwargs` after `verbose` and then when calling `read_raw_*` later, put `**kwargs` at the end. Also update docstring when you add it.